osgp/shared/shared/src/main/java/org/opensmartgridplatform/shared/application/scheduling/OsgpScheduler.java

```diff
@@ -127,6 +130,30 @@ public Trigger createJobTrigger(
         .build();
   }
 
+  /**
+   * Create a {@link Trigger} instance using a {@link JobDetail} instance and a delay in seconds.
```

Comment:
Delay could probably be any interval unit, not just seconds, as the interval unit is also passed as a parameter?
```suggestion
   * Create a {@link Trigger} instance using a {@link JobDetail} instance and a delay.
```

Reply:
Changed it